### ISSUE TITLE
Mixed content on browser based request

### ISSUE THREAD

**Issue by RongxinZhang** [Retsly/log#40]
On www, request from `http://127.0.0.1:5000` is blocked. Because of mixed Content error 

<img width="1394" alt="screen shot 2015-07-15 at 11 31 33 am" src="https://cloud.githubusercontent.com/assets/3782456/8706529/164294da-2ae5-11e5-9108-198e3590f75c.png">

**Comment by lvivier** [comment]
@slajax suggested we export a middleware instead of having logd host its own HTTP server.

- [ ] Remove HTTP server from bin/logd
- [ ] Rewrite [express app](https://github.com/Retsly/log/blob/master/lib/http.js) to communicate with logd rather than emit an event
- [ ] Expose app so it can be used as a middleware

Going in this direction means complicating the design somewhat; an express app can't just use a `Logger` instance for obvious reasons (ctx is fixed, ip and timestamp are calculated internally, etc) but we could move the log message object creation responsibility out of the client and share the logic between the client and the rewritten express app.

**Comment by slajax** [comment]
@lvivier if you think there is a better way, feel free to disagree with me. I didn't have a chance to give this much thought but it seemed like it would be easier to surface it through the www or api app then mount a separate service to nginx.

**Comment by lvivier** [comment]
@slajax I agree that it would be easier from a devops standpoint and I think we should do it…just trying to think thru it :+1:
